docs/pages/enroll-resources/server-access/guides/recording-proxy-mode.mdx

```diff
@@ -15,15 +16,6 @@ when gradually transitioning large server fleets to Teleport.
   ![Teleport OpenSSH Recording Proxy](../../../../img/server-access/openssh-proxy.svg)
 </Figure>
 
-<Notice type="warning">
```

Comment:
Why are we removing this info? Isn't it still true?

Reply:
It's not required to get the recording anymore. If we want to encourage moving to using ssh service then can modify this.

Reply:
I don't think we should remove this. You cannot enable proxy recording mode on cloud.